### PR TITLE
Log GnuPG home directory

### PR DESCRIPTION
**Pull request by DemiMarie**
This allows determining why a directory does not exist.

### CODE REVIEW

**Comment by marmarek** [comment]
Please squash `fixup!` commit to its base.

@pietrushnic was this useful for debugging?

**Comment by pietrushnic** [comment]
@marmarek no, finally I didn't needed since the problem was that I didn't imported  correct public key on GPG VM client side.

File: splitgpg2/__init__.py [hunk 1]
```diff
@@ -322,6 +328,12 @@ def load_config(self, config) -> None:
         for option in config:
             if option not in supported_options:
                 self.log.warning('Unsupported config option: %s', option)
+        self.log.info('Using GnuPG home directory %s', self.gnupghome)
+        try:
+            subprocess.check_call(('mkdir', '-pm0700', '--', self.gnupghome))
```
Comment (marmarek) [inline splitgpg2/__init__.py:333]: `os.makedirs`?

**Comment by marmarek** [comment]
This is now obsolete as #12 is merged

**Pull request closed without merging**